src/tensorlake/function_executor/handlers/run_function/response_helper.py

```diff
@@ -64,6 +66,18 @@ def failure_response(
             success=False,
         )
 
+        response.failure.scope = ProtoFailureScope.FAILURE_SCOPE_TASK
+        if failure.scope == FailureScope.Invocation:
+            response.failure.scope = ProtoFailureScope.FAILURE_SCOPE_INVOCATION
+        elif failure.scope == FailureScope.Graph:
+            response.failure.scope = ProtoFailureScope.FAILURE_SCOPE_GRAPH
+
+        response.failure.cls = failure.cls
```

Comment:
Why do we need to save all this human readable data in response object if we already saved it to stderr and customers will see it in stderr?